usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -125,3 +138,23 @@ def handle_subawards(filters: dict) -> dict:
         results["subcontracts"] = sum([sub["count"] for sub in queryset if sub["award_type"] == "procurement"])
 
         return results
+
+    def query_elasticsearch(self, filters) -> list:
+        filter_query = QueryWithFilters.generate_elasticsearch_query(filters, query_type="awards")
```

Comment:
Seems odd to run a search for each award type count. Couldn't this be redesigned as a single aggregation?

Reply:
I do realize the postgres version is running multiple queries on each type, but that was changed a few months ago for performance reasons. Elasticsearch as been running count aggregations for Keyword Search for a while without any noticeable issues

Reply:
Can you point me to where the keyword search uses aggregations for the counts? it looks to me like it's doing the same thing based off `spending_by_transaction_count` in `usaspending_api/search/v2/elasticsearch_helper.py`

Reply:
Oh wow, you're right......

Well, nix the comment I made about Keyword Search aggregations. Thinking about the original setup, there was an index for each FY + award type. Each count by type was running on 12 indexes. Now there is one per data type, so I expect a single aggregation query to be easier to implement.

Reply:
Switched to using a single aggregation query.